CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [0.26.0](https://github.com/redhat-developer/vscode-xml/milestone/34?closed=1) (July 3, 2023)
+
+### Enhancements
+
+ * Add support for code actions that are not associated with any diagnostics (quick assist). See [#1516](https://github.com/eclipse/lemminx/issues/1516).
+ * Make the rename participants conform to the LSP, permitting to rename across different documents. See [#1521](https://github.com/eclipse/lemminx/issues/1521).
```

Comment:
I think you are right, I don't think there is any user-visible improvements from these changes in vscode-xml. I think we should remove these from the changelog.

Reply:
I've removed them.